File: bin/s.f90
#!/bin/ksh
COMPILING_FORTRAN=YES
. s.get_compiler_rules.dot
[[ -n $Verbose ]] && set -x
$F90C ${SourceFile} ${FC_options} ${FFLAGS} \
      $(s.prefix "${Dprefix}" ${DEFINES} ) \
      $(s.prefix "${Iprefix}" ${INCLUDES} ${EC_INCLUDE_PATH}) \
      $(s.prefix "${Lprefix}" ${LIBRARIES_PATH} ${EC_LD_LIBRARY_PATH}) \
      $(s.prefix "${lprefix}" ${LIBRARIES} ${SYSLIBS} ) \
      "$@"

File: bin/s.cc
#!/bin/ksh
export COMPILING_C=yes
. s.get_compiler_rules.dot
[[ -n $Verbose ]] && set -x
if [[ -n $WILL_LINK ]]
then
  $CC ${SourceFile} $CC_options ${CFLAGS} \
      $(s.prefix "" ${DEFINES} ) \
      $(s.prefix "-I" ${INCLUDES} ${EC_INCLUDE_PATH}) \
      $(s.prefix "-L" ${LIBRARIES_PATH} ${EC_LD_LIBRARY_PATH}) \
      $(s.prefix "-l" ${LIBRARIES} ) \
      "$@"
else
  $CC ${SourceFile} ${CC_options_NOLD:-${CC_options}} ${CFLAGS} \
      $(s.prefix "" ${DEFINES} ) \
      $(s.prefix "-I" ${INCLUDES} ${EC_INCLUDE_PATH}) \
      "$@"
fi